doc/contributing/api-documentation.md

```diff
@@ -302,3 +301,5 @@ mutate the data and appends it to the final JSON object.
 
 For a more in-depth information we recommend to refer to the `json.mjs` file as
 it contains a lot of comments.
+
+[API Source]: https://github.com/nodejs/node/tree/main/doc/api
```

Comment:
Usually, the text content is used as reference.

Reply:
Fixed